// Copyright 2024 Google LLC
//
// Licensed under the Apache License, Version 2.0 (the "License");
// you may not use this file except in compliance with the License.
// You may obtain a copy of the License at
//
//      http://www.apache.org/licenses/LICENSE-2.0
//
// Unless required by applicable law or agreed to in writing, software
// distributed under the License is distributed on an "AS IS" BASIS,
// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
// See the License for the specific language governing permissions and
// limitations under the License.

syntax = "proto3";

package privacysandbox.apis.parc.v0;

// Request to get a parameter.
message GetParameterRequest {
  // The given name of the parameter to be retrieved.
  string parameter_name = 1;
}

// Response of getting a parameter.
message GetParameterResponse {
  // Returned parameter value.
  string parameter_value = 1;
}
